src/ToDo.js

```diff
@@ -11,23 +11,25 @@ class ToDo extends React.Component {
     this.state = {
       tasks: {},
```

Comment:
It should be an array
```suggestion
      tasks: {
        // id: 1, or Date.now(), or random-string https://www.npmjs.com/package/randomstring
        // title: 'Todo Title',
        // completed: false,
      },
```